src/pytest_postgresql/factories.py

```diff
@@ -20,20 +20,33 @@
 import os.path
 import platform
 import subprocess
+from contextlib import contextmanager
 from tempfile import gettempdir
+from types import TracebackType
+from typing import Union, Type, Optional, TypeVar, Any
+from warnings import warn
 
 import pytest
+from packaging.version import Version
 from pkg_resources import parse_version
 
 try:
     import psycopg2
+    try:
+        from psycopg2._psycopg import cursor
+    except ImportError:
+        from psycopg2cffi._impl.cursor import Cursor as cursor
 except ImportError:
     psycopg2 = False
+    cursor = Any  # pylint:disable=invalid-name # if there's no postgres, just go with the flow.
```

Comment:
Not needed if only for typing.

Reply:
if there's no psycopg2 installed, then if there's not psycopg2 installed, we'll get `NameError: name 'cursor' is not defined` error, instead of proper ImprotError like message in proper place